snappy_wrappers/wrappers/pvactools/combining/comb_rna.py

```diff
@@ -0,0 +1,530 @@
+                args.input_vcf
+            )
+        )
+    elif is_multi_sample and args.sample_name not in vcf_reader.header.samples.names:
+        vcf_reader.close()
+        raise Exception(
+            "ERROR: VCF {} does not contain a sample column for sample {}.".format(
+                args.input_vcf, args.sample_name
+            )
+        )
+    if "CSQ" not in vcf_reader.header.info_ids():
+        vcf_reader.close()
+        raise Exception(
+            "ERROR: VCF {} is not VEP-annotated. Please annotate the VCF with VEP before running this tool.".format(
+                args.input_vcf
+            )
+        )
+    if args.mode == "gene" and "GX" in vcf_reader.header.format_ids():
+        vcf_reader.close()
+        raise Exception(
+            "ERROR: VCF {} is already gene expression annotated. GX format header already exists.".format(
+                args.input_vcf
+            )
+        )
+    elif args.mode == "transcript" and "TX" in vcf_reader.header.format_ids():
+        vcf_reader.close()
+        raise Exception(
+            "ERROR: VCF {} is already transcript expression annotated. TX format header already exists.".format(
+                args.input_vcf
+            )
+        )
+    # snv part
+
+    return vcf_reader, rna_vcf_reader, is_multi_sample
+
+
+########################## Adding extra fields into Format ##########################
+def create_vcf_writer(args, vcf_reader):
+    (head, sep, tail) = args.input_vcf.rpartition(".vcf")
+    new_header = vcf_reader.header.copy()
+    if args.mode == "gene":
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "GX"),
+                    ("Number", "."),
+                    ("Type", "Float"),
+                    ("Description", "Gene Expressions"),
+                ]
+            )
+        )
+        output_file = ("").join([head, ".gx.vcf", tail])
+    elif args.mode == "transcript":
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "TX"),
+                    ("Number", "."),
+                    ("Type", "Float"),
+                    ("Description", "Transcript Expressions"),
+                ]
+            )
+        )
+        output_file = ("").join([head, ".tx.vcf", tail])
+    if args.rna_vcf:
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "RAD"),
+                    ("Number", "."),
+                    ("Type", "Integer"),
+                    (
+                        "Description",
+                        "Allelic depths from mRNA sequencing data of the same sample",
+                    ),
+                ]
+            )
+        )
+        new_header.add_format_line(
+            vcfpy.OrderedDict(
+                [
+                    ("ID", "ROT"),
+                    ("Number", "."),
```

Comment:
Should the number be `1` here? As I understand, this is the sum of the pileup at that position, which alt alleles are different from the one found in DNA.